unitpackage/electrochemistry/reference_electrodes.py

```diff
@@ -0,0 +1,278 @@
+        "temperatureDependence": {
+            "formula": "E = 0.63495 - 781.44E-6 * T - 426,89E-9 * T**2",
+            "comment": "E is in V and T in °C. Equation is valid in the range of 0°C to 60°C",
+            "doi": "https://doi.org/10.1021/ja01304a009",
+        },
+        "source": {"doi": "https://doi.org/10.1039/FT9949001875"},
+        "alternativeValues": [
+            {"value": 0.61587, "source": "https://doi.org/10.1039/TF9605601172"},
+            {"value": 0.61515, "source": "https://doi.org/10.1021/ja01304a009"},
+            {"value": 0.6125, "source": "https://doi.org/10.1039/TF9656102050"},
+            {"value": 0.61236, "source": "https://doi.org/10.1039/FT9949001875"},
+            {"value": 0.61544, "source": "https://doi.org/10.1007/BF00973518"},
+        ],
+    },
+    "MSE-sat": {
+        "value": 654,
+        "unit": "V",
+        "vs": "SHE",
+        "source": "Internally used reference value.",
+    },
+    "RHE": {
+        "value": 0.000,
+        "unit": "V",
+        "vs": "SHE",
+        "source": "Nernst equation, 25 °C.",
+    },
+}
+
+
+@dataclass(frozen=True)
+class ReferenceElectrode:  # pylint: disable=too-many-instance-attributes
+    """
+    Represents a electrochemical reference electrode.
+
+    Attributes
+    ----------
+    name : str
+        Common name of the reference electrode (e.g., 'Ag/AgCl-sat', 'SHE').
+    fullName: str
+         Spelled out name.
+    value : float
+        Potential of the electrode relative to the standard hydrogen electrode (SHE), in volts.
+    unit : str
+        Unit of the potential, typically 'V'.
+    vs : str
+        The reference scale against which the value is reported.
+    source : str
+        Bibliographic or textual source for the potential value.
+    alternativeValues : list
+        A list of alternative values including references for these values.
+    choice : str
+        Choice for choosing the specific reference value.
+    temperatureDependence : dict
+        A dict providing information about the temperature dependence of the reference electrode.
+
+    Examples
+    --------
+    Accessing reference electrode data:
+
+    >>> from unitpackage.electrochemistry.reference_electrodes import ReferenceElectrodes
+    >>> ReferenceElectrodes["Ag/AgCl-sat"] # doctest: +NORMALIZE_WHITESPACE
+    ReferenceElectrode(name='Ag/AgCl-sat', value=0.197, unit='V',
+    vs='SHE', source={'isbn': '978-1119334064'}, alias=None,
+    fullName=None, temperatureDependence=None, choice=None, alternativeValues=None)
+
+    Converting between reference scales:
+
+    >>> ReferenceElectrodes.convert(0.55, "Ag/AgCl-sat", "SHE")
+    0.35300000000000004
+
+    >>> ReferenceElectrodes.convert(ref_from="SHE", ref_to="RHE", ph=5)
+    -0.2955
+    """
+
+    name: str
+    value: float
+    unit: str = "V"
+    vs: str = "SHE"
+    source: str = ""
+    alias: str | None = None
+    fullName: str | None = None  # pylint: disable=invalid-name
```

Comment:
pylint wants snake_case naming convention. I assume this can be solved with pydantic, but I have not used that before.